ui/app/manager/src/pages/page-users.ts

```diff
@@ -395,10 +402,7 @@ class PageUsers<S extends AppStateKeyed> extends Page<S> {
                                     this._users = [...this._users, {enabled: true}];
                                 }}">
                                     <td colspan="100%">
-                                        <or-mwc-input class="button" .type="${InputType.BUTTON}"
-                                                      .label="${i18next.t("add")} ${i18next.t("user")}"
-                                                      icon="plus">
-                                        </or-mwc-input>
+                                        <a class="button"><or-icon icon="plus"></or-icon>${i18next.t("add")} ${i18next.t("user")}</a>
```

Comment:
Some browsers might have a problem with `href` not being present in an anchor element. Check if `href="#"` works.